classes/class-pmpro-admin-activity-email.php

```diff
@@ -346,7 +346,7 @@ public function sendAdminActivity( $frequency = '', $recipient = null ) {
 									?>
 									</td>
 									<td width="40%" style="background-color:#F1F1F1;padding:15px;">
-										<p style="margin:0px;padding:0px;text-align:center;"><a style="color:#1A688B;" href="https://www.paidmembershipspro.com" target="_blank"><img style="width:100px;height:100px;" src="<?php echo esc_url( plugins_url( 'images/Paid-Memberships-Pro_icon.png', PMPRO_BASE_FILE ) ); ?>" alt="<?php esc_html_e( 'Paid Memberships Pro', 'paid-memberships-pro' ); ?>" /></a></p>
+										<p style="margin:0px;padding:0px;text-align:center;"><a style="color:#1A688B;" href="https://www.paidmembershipspro.com" target="_blank"><img style="width:100px;height:100px;" src="<?php echo esc_url( plugins_url( 'images/Paid-Memberships-Pro_icon.png', PMPRO_BASE_FILE ) ); ?>" alt="<?php esc_attr_e( 'Paid Memberships Pro', 'paid-memberships-pro' ); ?>" /></a></p>
 										<p style="margin:0px 0px 15px 0px;padding:0px;"><a style="color:#1A688B;" href="https://www.paidmembershipspro.com/support/" target="_blank"><?php esc_html_e( 'Get Support', 'paid-memberships-pro' ); ?></a></p>
 										<p style="margin:0px 0px 15px 0px;padding:0px;"><a style="color:#1A688B;" href="https://twitter.com/pmproplugin" target="_blank"><?php esc_html_e( 'Follow @pmproplugin on Twitter', 'paid-memberships-pro' ); ?></a></p>
```

Comment:
Since this is an external link opening as the `_blank` target , we need the `rel` set for noopener/noreferrer for [security purposes](https://web.dev/external-anchors-use-rel-noopener/).

```suggestion
										<p style="margin:0px 0px 15px 0px;padding:0px;"><a style="color:#1A688B;" href="https://twitter.com/pmproplugin" target="_blank" rel="noopener noreferrer"><?php esc_html_e( 'Follow @pmproplugin on Twitter', 'paid-memberships-pro' ); ?></a></p>
```